web/groups/hlt.json

```
@@ -504,9 +510,11 @@
   "PixelJetPuId|": "Jets/MET",
   "PixelTrackFilterByKinematicsProducer|": "Pixels",
   "PixelTrackProducerFromSoA|": "Pixels|Conversion",
+  "PixelTrackProducerFromSoAPhase1|": "Pixels|Conversion",
   "PixelTrackProducerFromSoAPhase2|" : "Pixels|Conversion",
   "PixelTrackProducer|": "Pixels",
   "PixelTrackSoAFromCUDA|": "Pixels|Host copy",
+  "PixelTrackSoAFromCUDAPhase1|": "Pixels|Host copy",
   "PixelTrackSoAFromCUDAPhase2|" : "Host copy",
```

This looks like an old bug :-/
It should have been
```suggestion
  "PixelTrackSoAFromCUDAPhase2|" : "Pixels|Host copy",
```

Could you fix it ?

---

Done. Changed as suggested.